azkaban-exec-server/src/test/java/azkaban/execapp/event/RemoteFlowWatcherTest.java

```diff
@@ -54,17 +55,21 @@
   public void setUp() throws Exception {
     this.jobtypeManager =
         new JobTypeManager(null, null, this.getClass().getClassLoader());
-    this.jobtypeManager.getJobTypePluginSet().addPluginClass("java", JavaJob.class);
+    this.jobtypeManager.getJobTypePluginSet().addPluginClass("test", InteractiveTestJob.class);
     this.fakeProjectLoader = new MockProjectLoader(this.workingDir);
+    Utils.initServiceProvider();
+    JmxJobMBeanManager.getInstance().initialize(new Props());
+    InteractiveTestJob.setQuickSuccess(true);
   }
 
   @After
   public void tearDown() throws IOException {
+    InteractiveTestJob.resetQuickSuccess();
   }
 
   public File setupDirectory() throws IOException {
     System.out.println("Create temp dir");
-    final File workingDir = new File("_AzkabanTestDir_" + this.dirVal);
+    workingDir = new File("build/tmp/_AzkabanTestDir_" + this.dirVal);
```

Comment:
Could you convert it to tempDir junit rule too?

Reply:
I think this is already fixed in the pre-requisite PR that enables the RemoteFlowWatcherTest. Once that is merged I will update this PR.